src/Microsoft.AspNetCore.SignalR/HubRouteBuilder.cs

```diff
@@ -19,10 +20,10 @@ public HubRouteBuilder(SocketRouteBuilder routes)
 
         public void MapHub<THub>(string path) where THub : Hub
         {
-            MapHub<THub>(path, socketOptions: null);
+            MapHub<THub>(new PathString(path), socketOptions: null);
         }
 
-        public void MapHub<THub>(string path, Action<HttpSocketOptions> socketOptions) where THub : Hub
+        public void MapHub<THub>(PathString path, Action<HttpSocketOptions> socketOptions) where THub : Hub
```

Comment:
Let's also have an overload that just takes just a `PathString` (setting `socketOptions` to null).